paragon/blocks/_authn.scss

```diff
@@ -133,6 +135,131 @@
 
             h1.data-hj-suppress {
                 max-width: none;
+                color: $white;
+            }
+        }
+
+        .pp-page-content {
+            @include media-breakpoint-down(xs) {
+                margin: 1.5rem .5rem !important;
+            }
+        }
+
+        h1.welcome-to-platform {
+            color: $white;
+            margin-bottom: 40px;
+        }
+
+        .organization-select-info {
+            color: #888;
+            font-size: 14px;
+        }
+
+        .authn-organization-form {
+            line-height: normal;
+
+            .pgn__form-control-decorator-group {
+                margin-right: 0 !important;
+            }
+
+            .pgn__form-label {
+                margin-bottom: 1rem;
+                line-height: 1.4;
+                font-size: 15px;
+            }
+
+            #is_organization_registered .pgn__form-label {
+                @media screen and (max-width: 450px) {
```

Comment:
[question] can we use `@include media-breakpoint-down(xs) {}` instead of creation custom media query?

Reply:
No, we need super custom media query for out cases